CHANGELOG.md

```diff
@@ -10,7 +10,7 @@
 
 ### Enhancements
 
-* Lorem ipsum.
+* Implemented `realm::util::File::is_dir`, `realm::util::File::resolve`, and `realm::util::DirScanner` on Windows.
```

Comment:
This line is much longer than the rest of the lines in this file.

In sync we stick to 80 columns for the CHANGELOG, don't know if Core follows the same pattern.